packages/data-context/src/actions/ProjectActions.ts

```diff
@@ -85,7 +85,6 @@ export class ProjectActions {
 
   async setCurrentProject (projectRoot: string) {
     await this.clearCurrentProject()
```

Comment:
`this.clearCurrentProject()` already calls `this.ctx.lifecycleManager.clearCurrentProject()`: https://github.com/cypress-io/cypress/pull/19580/files#diff-d36f6626bf805d1bbbfba4786ca3a518f0083613a9814fd037c5592808949606L56